dev-java/iso-relax/iso-relax-20050331-r6.ebuild

```diff
@@ -10,30 +10,23 @@ inherit java-pkg-2 java-pkg-simple
 DESCRIPTION="Interfaces useful for applications which support RELAX Core"
 HOMEPAGE="http://www.xml.gr.jp/relax/"
 SRC_URI="mirror://gentoo/${P}.tar.bz2"
+S="${WORKDIR}/${P}"
 
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="amd64 ~arm arm64 ppc64 x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-solaris"
 
-RESTRICT="test"
-
-CDEPEND="dev-java/ant-core:0"
-
 DEPEND="
-	${CDEPEND}
-	>=virtual/jdk-1.8:*"
-
-RDEPEND="
-	${CDEPEND}
-	>=virtual/jre-1.8:*"
-
-S="${WORKDIR}/${P}"
+	dev-java/ant:0
```

Comment:
this package also seems to need `ant` during runtime:
```
$ grep -i \\.ant * -r
src/org/iso_relax/ant/ErrorHandlerImpl.java:package org.iso_relax.ant;
src/org/iso_relax/ant/ErrorHandlerImpl.java:import org.apache.tools.ant.Project;
src/org/iso_relax/ant/ErrorHandlerImpl.java:import org.apache.tools.ant.Task;
src/org/iso_relax/ant/JARVTask.java:package org.iso_relax.ant;
src/org/iso_relax/ant/JARVTask.java:import org.apache.tools.ant.BuildException;
src/org/iso_relax/ant/JARVTask.java:import org.apache.tools.ant.DirectoryScanner;
src/org/iso_relax/ant/JARVTask.java:import org.apache.tools.ant.Project;
src/org/iso_relax/ant/JARVTask.java:import org.apache.tools.ant.Task;
src/org/iso_relax/ant/JARVTask.java:import org.apache.tools.ant.types.FileSet;
```

Reply:
done.